src/main/resources/application.yaml

```diff
@@ -46,6 +52,8 @@ security:
     - "/swagger-resources/**"
     - "/v2/api-docs"
     - "/webjars/springfox-swagger-ui/**"
+  authorisedServices:
+    - "iac"
```

Comment:
iac? Isn't that an I&A thing?

Reply:
Yes it is.  I re-used iac as an authorized service since we already have a S2S Microservice set up.  Need to create one for our app group so we can use that instead along with a secret.